docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -2,20 +2,56 @@
 
--	**Larger batch size**<br>
-	Notice that the devices like GPU are doing better with larger batch size. While it is possible to set the batch size in the runtime using the OpenVINO Runtime API [ShapeInference feature](../../OV_Runtime_UG/ShapeInference.md).
+- **Larger batch size**<br>
+    Some devices, like GPU, achieve better results with larger batch sizes. In such cases,
+    it is possible to set the batch size using `--batch` option of Model Optimizer or the
+    OpenVINO Runtime API [ShapeInference feature](../../OV_Runtime_UG/ShapeInference.md).
 
--	**Resulting IR precision**<br>
-The resulting IR precision, for instance, `FP16` or `FP32`, directly affects performance. As CPU now supports `FP16` (while internally upscaling to `FP32` anyway) and because this is the best precision for a GPU target, you may want to always convert models to `FP16`. Notice that this is the only precision that Intel&reg; Movidius&trade; Myriad&trade; 2 and Intel&reg; Myriad&trade; X VPUs support.
+## When to Specify Layout
+
+Layout defines which dimensions are located in shape and can be specified
+for both inputs and outputs. For layout syntax check the [Layout API overview](../../OV_Runtime_UG/layout_overview.md). 
+To specify layout, you can use either `--layout` or `--source_layout` followed by the layout value. 
+Additionally, if a model has more than one input or needs both input and output layouts specified,
+each layout needs to be given a name. Here are examples of the two cases:
+`NCHW` layout specified for a model with a single input:
+```
+mo --input_model /path/to/model --source_layout nchw
+mo --input_model /path/to/model --layout nchw
+```
+`NCHW` and `NC` layouts specified for a model with two inputs:
+```
+mo --input_model /path/to/model --source_layout name1(nchw),name2(nc)
+mo --input_model /path/to/model --layout name1(nchw),name2(nc)
+```
+Some preprocessing may require setting of input layouts, for example: batch setting,
+application of mean or scales, and reversing input channels (BGR<->RGB).
+
+## How to Change Layout of a Model 
+
+It is possible to change layout of the model. It may be needed if input data has different
+layout then model was trained on. There are 2 options that can be used to change layout of
+model inputs or outputs --layout and --target_layout. For example to change layout of the
+model with one input from `NHWC` to `NCHW`:
+```
+mo --input_model /path/to/model --source_layout nhwc --target_layout nchw
+mo --input_model /path/to/model --layout "nhwc->nchw"
+```
+Similarly, if model has multiple inputs or if it is required to change layout of output,
+names must be specified:
+```
+mo --input_model /path/to/model --source_layout name1(nhwc),name2(nc) --target_layout name1(nchw)
+mo --input_model /path/to/model --layout "name1(nhwc->nchw),name2(nc)"
+```
```

Comment:
```suggestion
Changing model layout may be necessary if it differs from the one presented by input data. 
Similarly to specifying layout, to change it using one of two commands: either `--layout` or `--target_layout`. 
Again, if a model has more than one input or needs both input and output layouts specified,
each layout needs to be given a name. Here are examples of the two cases:

`NHWC` layout of a single-input model changed to `NCHW`:
```
mo --input_model /path/to/model --source_layout nhwc --target_layout nchw
mo --input_model /path/to/model --layout "nhwc->nchw"
```

`NHWC` layout of a multi-input model changed to `NCHW`:
```
mo --input_model /path/to/model --source_layout name1(nhwc),name2(nc) --target_layout name1(nchw)
mo --input_model /path/to/model --layout "name1(nhwc->nchw),name2(nc)"
```
```